tests/integration/test_array_creation.py

```diff
@@ -137,21 +137,26 @@ def test_full_bad_filled_value(self):
     (np.arange(24).reshape(4, 3, 2), "f4"),
 ]
 LIKE_FUNCTIONS = ("zeros_like", "ones_like")
+SHAPE_ARG = (None, (-1,), (1, -1))
 
 
 @pytest.mark.parametrize("x_np,dtype", DATA_ARGS)
-def test_empty_like(x_np, dtype):
+@pytest.mark.parametrize("shape", SHAPE_ARG)
+def test_empty_like(x_np, dtype, shape):
+    shape = shape if shape is None else x_np.reshape(shape).shape
     x = num.array(x_np)
-    xfl = num.empty_like(x, dtype=dtype)
-    yfl = np.empty_like(x_np, dtype=dtype)
+    xfl = num.empty_like(x, dtype=dtype, shape=shape)
+    yfl = np.empty_like(x_np, dtype=dtype, shape=shape)
 
     assert xfl.shape == yfl.shape
     assert xfl.dtype == yfl.dtype
 
 
 @pytest.mark.parametrize("x_np,dtype", DATA_ARGS)
 @pytest.mark.parametrize("fn", LIKE_FUNCTIONS)
-def test_func_like(fn, x_np, dtype):
+@pytest.mark.parametrize("shape", SHAPE_ARG)
+def test_func_like(fn, x_np, dtype, shape):
+    shape = shape if shape is None else x_np.reshape(shape).shape
```

Comment:
The `shape` is not actually used in this test (and the one below?)

Reply:
Oops, forgot to commit, fixed.